src/Microsoft.Data.Sqlite/SqliteCommand.cs

```diff
@@ -111,11 +114,16 @@ public new virtual SqliteDataReader ExecuteReader(CommandBehavior behavior)
             do
             {
                 Sqlite3StmtHandle stmt;
-                var rc = NativeMethods.sqlite3_prepare_v2(
-                    Connection.DbHandle,
-                    tail,
-                    out stmt,
-                    out tail);
+                var rc = NativeMethods.sqlite3_prepare_v2_blocking(
```

Comment:
Yes, it will occasionally return busy. I couldn't find a case of it returning locked.